setup.py

```diff
@@ -1,62 +1,62 @@
-import setuptools
-
-py_classifiers = [f"Programming Language :: Python :: {version}" for version in ["3.7"]]
-
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from setuptools import find_packages, setup
 
 extras_require = {
     "test": [
-        "factory-boy>=2.12.0,<3.0.0",
         "pytest>=4.6.3,<5.0.0",
+        "pytest-xdist>=1.30.0,<2",
         "pytest-asyncio>=0.10.0,<1.0.0",
-        "pytest-xdist>=1.30.0",
+        "factory-boy>=2.12.0,<3.0.0",
     ],
     "lint": [
-        "mypy>=0.701,<1.0",
+        "flake8==3.7.9",  # flake8 is not semver: it has added new warnings at minor releases
+        "isort==4.3.21",
+        "mypy==0.740",  # flake8 is not semver: it has added new warnings at minor releases
```

Comment:
spurious comment?

Reply:
Ah thanks, I forgot to reword the comment (roughly the same thing applies to mypy).